# NOTE: Copy this file.
# cls_color & cls_color_thres support (H, S, V) tuples too.

filter_method: hsv
filter_common_thres: 120
filter_common_s_range: [0, 255]
filter_common_v_range: [0, 255]
filter_hsv_lower: [90, 0, 0]
filter_hsv_upper: [235, 255, 255]
pp_open: 5
pp_close: 40
pp_blur: 0
opi_small: 0.0005
opi_large: 0.7
cls_color_flare: 45
cls_color_thres_flare: 90
cls_sol_gate: 0.15
cls_sol_thres_gate: 0.15
cls_color_blue: 100
cls_color_thres_blue: 0
cls_color_red: 100
cls_color_thres_red: 0
cls_color_yellow: 100
cls_color_thres_yellow: 0
